### ISSUE TITLE
[Bug]: Version bump PR for functional test repo failed to update pack-lock.json

### ISSUE THREAD

**Issue by zelinh** [opensearch-project/opensearch-build#4638]
### Describe the bug

The auto-generated version bump PR for functional test repo didn't include `package-lock.json` file. 
e.g. https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/1216

It should have increment the version in `package-lock.json` here https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/45396abd554474e1eca18f20bb77bf42ee11fafa/package-lock.json#L3 
along with `package.json`. 

Seems like we missed it here.
https://github.com/opensearch-project/opensearch-build/blob/90352da641556ad9c900e8a5f9149fd836556319/.github/workflows/osd-increment-plugin-versions.yml#L95-L99

### To reproduce

Run the current GHA `Increment OpenSearch Dashboards Plugins Version`

**Comment by prudhvigodithi** [comment]
Hey @zelinh that is done on purpose as AFAIK for FT repo we dont `package-lock.json` file, here are some sample old PR's that only increments `package.json`
https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/1206/files 
https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/1121
Adding @AMoo-Miki to confirm.

@bbarani @Divyaasm

**Comment by AMoo-Miki** [comment]
Yeah; we shouldn't _need_ to bump it but it could help with tracking if we ever need to do so (which we have never needed to).

**Comment by rishabh6788** [comment]
Do we have a consensus on whether it should or should not be included? @AMoo-Miki

**Comment by AMoo-Miki** [comment]
Thinking about it more, we should keep the versions in the lock file and the manifest in sync.

**Comment by prudhvigodithi** [comment]
There is some discussion in past https://github.com/opensearch-project/opensearch-build/issues/3856 related to `yarn.lock` file. If just adding `package-lock.json` is required then:

- Ensure that the lock file is updated as part of the version increment plugin helper (`node ../../scripts/plugin_helpers version`).
- Add the file to git commit https://github.com/opensearch-project/opensearch-build/blob/main/.github/workflows/osd-increment-plugin-versions.yml#L193.

Thank you 
@peterzhuamazon @getsaurabh02

**Comment by Divyaasm** [comment]
The package-lock.json is not being updated currently check [here](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/30639d2ee16569cd73f3f5891ab850406a06b9cd/package-lock.json#L3). Do we still need to automate this?
